lab-andrew/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
This is a great start but your watch task is not being declared properly and is causing your gulpfile to not run well.  The mocha and linter tasks look good though.